src/widgets/WebexMeetingWidget.jsx

```diff
@@ -1,8 +1,57 @@
-import React from 'react';
-import webex from 'webex';
+import React, {useState, useEffect, Component} from 'react';
+import PropTypes from 'prop-types';
+import Webex from 'webex';
 import {WebexMeeting, WebexDataProvider} from '@webex/components';
 import WebexSDKAdapter from '@webex/sdk-component-adapter';
 
-export default function WebexMeetingWidget() {
-  return <h1>Webex Meeting Widget!</h1>;
+import '@webex/components/dist/webexComponents.css';
+import './WebexMeetingWidget.css';
+
+/**
+ * Webex meeting widget displays the default Webex meeting experience.
+ *
+ * @param {string} props.meetingDestination  ID of the virtual meeting location
+ * @param {string} props.accessToken        access token to create the webex instance with
+ * @returns {Object} JSX of the component
+ */
+export default class WebexMeetingWidget extends Component {
```

Comment:
Can we add snapshot tests?

Reply:
test environment is not setup yet. that will be outside of this PR scope.

Reply:
Why wasn't the test environment part of the repo setup?

Reply:
we decided to go with our new testing process for this repo which needs more efforts.